$NetBSD: patch-if,v 1.1 2008/01/05 15:15:01 rillig Exp $

if.c:130: error: static declaration of 'if_change_bits' follows non-static declaration
if.h:82: error: previous declaration of 'if_change_bits' was here

--- src/if.h.orig	1999-02-18 00:30:22.000000000 +0100
+++ src/if.h	2008-01-05 16:09:39.000000000 +0100
@@ -79,9 +79,6 @@
  * Physical interface information
  */
 
-extern const bits if_change_bits[];             /* Interface flag bits */
-extern const bits if_proto_bits[];
-
 struct _if_link {
 	struct _if_link *ifl_forw;
 	struct _if_link *ifl_back;
